backend/core/infra/query/master_book_impl.go

```
@@ -58,3 +59,13 @@ func (m *masterBook) Search(ctx context.Context, query query.MasterBookSearchQue
 	}
 	return masterBooks, nil
 }
+
+func (m *masterBook) GetByID(ctx context.Context, query query.MasterBookGetQuery, orFail bool) (*model.MasterBook, error) {
+	// TODO:実装
```

golangci-lint (2.2.2)

core/infra/query/master_book_impl.go:64: Line contains TODO/BUG/FIXME: "TODO:実装"

(godox)